posthog-android/src/main/java/com/posthog/android/replay/PostHogMaskModifier.kt

```diff
@@ -0,0 +1,20 @@
+package com.posthog.android.replay
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.SemanticsPropertyKey
+import androidx.compose.ui.semantics.semantics
+
+public object PostHogMaskModifier {
+    internal val PostHogReplayMask = SemanticsPropertyKey<Boolean>("ph-no-capture")
+
+    /**
+     * Marks the element as not to be captured by PostHog Session Replay.
+     */
+    public fun Modifier.postHogMaskReplay(): Modifier {
```

Comment:
```suggestion
    public fun Modifier. postHogMask(isEnabled: Boolean = true): Modifier {
```

So that people can use the modifier using a flag instead of doing if/else when enabling or not this behavior.

Reply:
If its `false`, then it should not be masked at all even if the other controls are enabled.